lib/simpleAuthRequester.js

```diff
@@ -0,0 +1,31 @@
+const request = require('request')
```

Comment:
I like `simpleAuthRequester`, but I think we should refactor, it overlaps with `requester`. We should use `simpleAuthRequester` everywhere.

Reply:
That makes a lot of sense. Would you please do the refactor to reduce the back and forth on this? 
Thanks.